dockers/docker-lldp-sv2/lldpd.conf.j2

```diff
@@ -1,3 +1,9 @@
 {% if MGMT_INTERFACE %}
-configure ports eth0 lldp portidsubtype local {{ MGMT_INTERFACE.keys()[0][0] }}
+{# If MGMT port alias is available, use it for port ID subtype, otherwise use port name #}
```

Comment:
should we do this in lldpmgrd or in this template?

Reply:
We could move MGMT port configuration into lldpmgrd to consolidate the configuration logic. I just updated the existing template here to make this a quick and minor change.